packages/near-fast-auth-signer/src/components/SignMultichain/schema.ts

```diff
@@ -0,0 +1,6 @@
+import { BorshSchema } from 'borsher';
+
+export const derivationPathSchema = BorshSchema.Struct({
```

Comment:
I think this is important to be on the docs, so the caller knows the schema.

Reply:
It's ok if you don't do on this PR, but create a issue for this and we do the docs later.